Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.ts

```diff
@@ -73,7 +76,9 @@ export class CustomGroupComponent implements OnInit, OnDestroy {
     private alertService: AlertService,
     @Inject('IAccessControlConfigService')
     private configService: IAccessControlConfigService,
-    private userNameService: NameDisplayService
+    private userNameService: NameDisplayService,
+    @Inject('IAuthService')
```

Comment:
does the test file need to be updated to have this added as well

Reply:
That could be my issue. Running the tests locally as well is giving me a StaticInjectorError[IAuthService]: NullInjectorError: No provider for IAuthService

Reply:
Updated the test file, thanks Jason and Brian for your help with this.